app/brave_command_ids.h

```diff
@@ -137,6 +137,8 @@
 // Adblock (user-defined cosmetic filters)
 #define IDC_ADBLOCK_CONTEXT_BLOCK_ELEMENTS 56351
 
+#define IDC_OPEN_FULL_PAGE_CHAT 56352
```

Comment:
@fallaciousreasoning should this be grouped with the other AI_CHAT commands above?